src/components/global/Header.vue

```diff
@@ -146,6 +149,15 @@ export default class Header extends Vue {
 		this.$store.commit('setNotifications', []);
 		Vue.$axios.post(`/notifications/mark_all_as_deleted`);
 	}
+
+	private isAdminUser() {
```

Comment:
I think instead of doing this which will be called every time the header is rendered. It would be better to send admin information when a user log in and save that in local storage. That way its just one call and then the back end can handle enforce it.